src/test/java/athleticli/ui/ParserTest.java

```diff
@@ -37,4 +42,28 @@ void parseCommand_byeCommand_expectByeCommand() throws AthletiException {
         final String byeCommand = "bye";
         assertInstanceOf(ByeCommand.class, parseCommand(byeCommand));
     }
+
+    @Test
+    void parseCommand_addSleepCommand_expectAddSleepCommand() throws AthletiException {
+        final String addSleepCommandString = "add-sleep /start 10:00 PM /end 6:00 AM";
+        assertInstanceOf(AddSleepCommand.class, parseCommand(addSleepCommandString));
+    }
+
+    @Test
+    void parseCommand_editSleepCommand_expectEditSleepCommand() throws AthletiException {
+        final String editSleepCommandString = "edit-sleep 1 /start 10:00 PM /end 6:00 AM";
+        assertInstanceOf(EditSleepCommand.class, parseCommand(editSleepCommandString));
+    }
+
+    @Test
+    void parseCommand_deleteSleepCommand_expectDeleteSleepCommand() throws AthletiException {
```

Comment:
Great naming according to Coding Standards

Reply:
Thanks!